typescript/cli/src/tests/cosmosnative/commands/helpers.ts

```diff
@@ -0,0 +1,97 @@
+import { ProcessOutput, ProcessPromise } from 'zx';
```

Comment:
ditto here, like you could have all the common stuff in a single place and then only reimplement things that are difficult to generalize for both vm types